### PR TITLE
Add guide for monitoring a k8s cluster with Netdata

### PR DESCRIPTION
**Pull request by joelhans**
##### Summary

Title is fairly self-explanatory. My idea is that k8s users first use the install instructions I'm hoping to add with #9364, then hop over to this guide to explore the various charts that they can now access once they've installed the Helm chart. I'm hoping it's helpful to explain what each of these sections does and what the capabilities are for each, as the UX around k8s isn't very refined right now.

This guide depends on and links to the file added in #9364, hence the broken link-check CI action. As long as we merge #9364 first and rebase this PR on top of that, the links should resolve OK.

##### Component Name

area/docs

### CODE REVIEW

**Comment by joelhans** [comment]
@ncmans Made the fixes you recommended in the other PR here. We can continue the conversation on this PR until it's ready to be merged.

File: docs/guides/monitor/kubernetes-k8s-netdata.md [hunk 1]
```diff
@@ -0,0 +1,274 @@
...
+-->
+
+# Monitor a Kubernetes cluster with Netdata
+
+While Kubernetes (k8s) might simplify the way you deploy, scale, and load-balance your applications, not all clusters
+come with "batteries included" when it comes to monitoring. Doubly so for a monitoring stack that helps you actively
+troubleshoot issues with your cluster.
+
+Some k8s providers, like GKE (Google Kubernetes Engine), do deploy clusters bundled with monitoring capabilities, such
+as Google Stackdriver Monitoring. However, these pre-configured solutions might offer the depth of metrics,
```
Comment (cakrit) [inline docs/guides/monitor/kubernetes-k8s-netdata.md:14]: you mean "might not offer" instead of "might offer"?

**Comment by cakrit** [comment]
This is fine for now. As soon as we have auto-claiming in kubernetes and claiming MVP1 deployed in production, we need to extend this with additional things people are able to do using the cloud, that are of particular interest to kubernetes. The visibility per node isn't ideal for such environments.

File: docs/guides/monitor/kubernetes-k8s-netdata.md (hunk 1 above)
Comment (joelhans) [reply docs/guides/monitor/kubernetes-k8s-netdata.md]: Yes, thanks for the catch.

**Pull request closed without merging**